library/src/main/scala/scala/scalajs/js/annotation/JSExportDescendentClasses.scala

```diff
@@ -17,4 +17,11 @@ package scala.scalajs.js.annotation
  *
  *  @see [[http://www.scala-js.org/doc/export-to-javascript.html Export Scala.js APIs to JavaScript]]
  */
-class JSExportDescendentClasses extends scala.annotation.StaticAnnotation
+class JSExportDescendentClasses extends scala.annotation.StaticAnnotation {
+  /** Constructor that allows ignoring descendants that cannot be exported
+   *
+   *  @param ignoreInvalidDescendants If true, descendants that cannot be exported
+   *      are silently ignored.
+   */
+  def this(ignoreInvalidDescendants: Boolean) = this()
```

Comment:
Actually, the others are not really fixable, since we cannot express one ctor in terms of the other (except for having the empty name mean the default name). I propose I fix these and leave the others.